src/main/java/site/coach_coach/coach_coach_server/action/service/ActionService.java

```diff
@@ -27,6 +29,9 @@ public Long createAction(Long categoryId, Long userIdByJwt,
 		Category category = categoryRepository.findById(categoryId)
 			.orElseThrow(() -> new NotFoundException(ErrorMessage.NOT_FOUND_CATEGORY));
 
+		Optional.ofNullable(category.getRoutine())
+			.orElseThrow(() -> new NotFoundException(ErrorMessage.NOT_FOUND_ROUTINE));
+
```

Comment:
이 optional 비싸다고 들었거든요? 근데 너무 많이 쓰는 것 같아서 바꿔주면 좋을 것 같아요
```suggestion
		Routine routine = category.getRoutine();
        if (routine == null) {
            throw new NotFoundException(ErrorMessage.NOT_FOUND_ROUTINE);
        }
```

Reply:
안 비싸대요 ㅎㅎ